app/src/main/java/com/wut/indoornavigation/data/model/graph/Vertex.java

```diff
@@ -3,10 +3,20 @@
 
 import com.wut.indoornavigation.data.model.Point;
 
+import lombok.Builder;
 import lombok.Value;
 
+/**
+ * Vertex class
+ */
 @Value
+@Builder(toBuilder = true)
 public class Vertex {
     int id;
     Point position;
+
+    public Vertex(int id, Point coordinates) {
```

Comment:
chyba musze to zostawic bo w Mesh w ProcessCell jak poboje stworzyc wierzcholek to mam:
Error:(307, 29) error: Vertex(int,Point) is not public in Vertex; cannot be accessed from outside package

mimo ze klasa jest publiczna